scripts/cmd_args.py

```diff
@@ -105,9 +142,6 @@ def add_common_options(parser):
                    on either side of each interval. Use the same --pad \
                    value that was supplied to bw2h5.py when creating \
                    the h5 files for training and validation.")
-    parser.add('--transform', required=True, type=str, choices=['log', 'None'],
-               help='transformation to apply to\
-                           coverage tracks before training')
     parser.add('--layers', required=True, type=type_or_none_fn(str),
```

Comment:
Is it necessary to supply this again? We can use the names that were supplied with the `layersbw` option.